new-charts/tc-legend/tc-legend-ordinal.scss

```
@@ -3,10 +3,14 @@
   .tc-legend-ordinal__serie[#{$attr}="#{$value}"] .tc-legend-ordinal__serie-color {
     background-color: $color;
   }
-  // New legend
+  // legacy
   .tc-details-serie-container[#{$attr}="#{$value}"] .tc-details-serie-color__rect {
     background-color: $color;
   }
+  // new
+  .details__series .serie__color[data-label="#{$value}"] {
```

Si on suit ma recommendation : https://github.com/ToucanToco/tucana/pull/6120#pullrequestreview-360915254
Il faudra changer ce sélecteur
```suggestion
  .serie[data-label="#{$value}"] .serie__color {
```